main.js

```diff
@@ -0,0 +1,417 @@
+        },
+        "Black Coffee" : {
+            className : "images",
+            src : "Black Coffee.jpg",
+            alt : "Black Coffee",
+        },
+        "Green Tea" : {
+            className : "images",
+            src : "Green Tea.jpeg",
+            alt : "Green Tea",
+        },
+    },
+
+    "Smoothy" : {
+        id : "Smoothy",
+        "Tea" : {
+            className : "images",
+            src : "Tea.jpeg",
+            alt : "Tea",
+        },
+        "Black Tea" : {
+            className : "images",
+            src : "Black-Tea.jpeg",
+            alt : "Black Tea",
+        },
+        "Coffee" : {
+            className : "images",
+            src : "Coffee.jpg",
+            alt : "Coffee",
+        },
+        "Black Coffee" : {
+            className : "images",
+            src : "Black Coffee.jpg",
+            alt : "Black Coffee",
+        },
+        "Green Tea" : {
+            className : "images",
+            src : "Green Tea.jpeg",
+            alt : "Green Tea",
+        },
+    },
+
+    "Ice Cream" : {
+        id : "IceCream",
+        "Tea" : {
+            className : "images",
+            src : "Tea.jpeg",
+            alt : "Tea",
+        },
+        "Black Tea" : {
+            className : "images",
+            src : "Black-Tea.jpeg",
+            alt : "Black Tea",
+        },
+        "Coffee" : {
+            className : "images",
+            src : "Coffee.jpg",
+            alt : "Coffee",
+        },
+        "Black Coffee" : {
+            className : "images",
+            src : "Black Coffee.jpg",
+            alt : "Black Coffee",
+        },
+        "Green Tea" : {
+            className : "images",
+            src : "Green Tea.jpeg",
+            alt : "Green Tea",
+        },
+    },
+};
+
+let cart_map=new Map();
+let all_ord=[];
+
+// Model
+
+function removeQty(key){
+    cart_map.delete(key);
+}
+
+function subQty(key){
+    if(cart_map.get(key)=="1"){
+        removeQty(key);
+    }
+    else{
+        let temp_num=Number(cart_map.get(key))-1;
+        cart_map.set(key,temp_num);
+    }
+};
+
+function addQty(key){
+    if(cart_map.get(key) === undefined){
+        cart_map.set(key,"1");
+    }
+    else{
+        let temp_num=Number(cart_map.get(key))+1;
+        cart_map.set(key,temp_num);
+    }
+}
+
+function addOredrToPrevious(e){
+    let temp_obj={};
+    temp_obj.date=new Date();
+    temp_obj.id="u01"+all_ord.length.toString();
+    temp_obj.status="Pending";
+    for(let key of cart_map.keys()){
+        temp_obj[key]=cart_map.get(key);
+    }
+    all_ord.unshift(JSON.stringify(temp_obj));
+    cart_map=new Map();
+}
+
+// View
+
+document.querySelector('body').addEventListener("load",DisplayItem());
+
+function DisplayItem () {
+
+    let sec=document.getElementById("LeftContent");
+
+    for(let key in items){
+
+        let sub_sec=document.createElement("section");
+        sub_sec.className="Inline_Flex";
+
+        for(let j in items[key]){
```

Comment:
don't use i/j type variable names

Use explanatory names like item or sectionItem whichever is application